utils/dump_taxon_tree.pl

```diff
@@ -265,33 +269,34 @@ sub node_to_dynatree {
   
   if (@{$node->dba}) {
     foreach my $dba (@{$node->dba}) {
-      my $display_name = get_node_display_name($dba, $name);
+      my $meta = get_meta($dba, $name);
       push @output, {  
-        key   => ucfirst($dba->species),
-        title => $display_name
+        key   => $meta->{'prod_name'},
+        title => $meta->{'display_name'}
       };
     }
   }  
   
   return @output;
 }
 
-sub get_node_display_name {
+sub get_meta {
   # this subroutine should not be necessary; we expect each node to have core adaptors, including the MetaContainer,
   # but at the same time, we are a bit nervous to trust the Perl API entirely
   my ($node_dba, $fallback_name) = @_;
   my $meta_adaptor = Bio::EnsEMBL::Registry->get_adaptor( $node_dba->species, "core", "MetaContainer" );
+  my $meta = {};
+
   if ($meta_adaptor) {
-    my $display_name = $meta_adaptor->get_display_name();
+    $meta->{'display_name'} = $meta_adaptor->get_display_name();
+    $meta->{'prod_name'} = $meta_adaptor->single_value_by_key('species.production_name');
```

Comment:
can a species not have a display_name or production_name? should you have a default value return or even empty string?

Reply:
or probably put a warning in there or a message. i know it shouldn't come to this but just in case. And explain what it means not having a production name or display name.

Reply:
Both display name and production names are expected.
https://genomes-ebi.slack.com/archives/C0J0Z9F96/p1632220195024700

But anyway I will change my logic slightly different to accommodate your thoughts.